torchci/lib/benchmark/api_helper/compilers/helpers/common.ts

```diff
@@ -17,9 +17,12 @@ export function extractBackendSqlStyle(
 export function toQueryArch(device: string, arch: string) {
```

Comment:
Would be nice to have a comment of what this function does (and may be type annotate the return values, if this is possible in TS)